server/docker/docker-compose.yml

```diff
@@ -211,19 +211,27 @@ services:
     extends:
       file: common-services.yml
       service: test_builder
-    command: ["--spring.config.location=classpath:./,file:/etc/defold/extender/", "--spring.profiles.active=local-dev-app", "--extender.authentication.platforms=linux", "--extender.authentication.users=file:/etc/defold/users/testusers.txt"]
+    command: ["--spring.config.location=classpath:./,file:/etc/defold/extender/", "--spring.profiles.active=local-dev-app", "--extender.authentication.platforms=linux", "--extender.authentication.users=file:/etc/defold/users/testusers.txt", "--server.port=9001"]
     volumes:
       - ./../users:/etc/defold/users:ro
     expose:
-      - "9000"
+      - "9001"
     ports:
-      - "9000:9000"
+      - "9001:9001"
```

Comment:
Use 9001 port for authentication testing to avoid port collision during parallel running.